README.md

```diff
@@ -214,30 +380,11 @@ There are two use cases for deep linking that can be relevant here:
 
 In order for deep linking to work, there are a few configurations that are needed and these are no different from what are required for handling deep linking in general and [Apple documentation](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) on this can be followed in conjunction with the steps highlighted here:
 
-### Option 1: Modify Open Tracking
-
-If you plan to use universal links in your app for deep linking you will need to modify the push open tracking as described below:
-
-```swift
-extension AppDelegate: UNUserNotificationCenterDelegate {
-    func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
-        let handled = KlaviyoSDK().handle(notificationResponse: response, withCompletionHandler: completionHandler) { url in
-            print("deep link is ", url)
-        }
-        if !handled {
-           // not a klaviyo notification should be handled by other app code
-        }
-    }
-}
-```
-
-Note that the deep link handler will be called back on the main thread. If you want to handle uri schemes in addition to universal links you implement them as described below.
-
-### Option 2: Use URL Schemes
+##### Option 1: URI Schemes
```

Comment:
One thing I commented on the Android README when Evan was doing the redux over that side is that the title nesting is getting a little crazy at this level. I wonder if we should just remove anything around universal links and just talk about URL scheme setup steps so that we dont need to get 5-6 layers deep.

Cant really remember where we landed with universal links stuff and having that in the readme though. Maybe @mark-piana would be clearer on that front

Reply:
I feel like it's good information to have in the readme if we support it. I know we're not hosting anything on our side, but I think we'll get questions if we remove information about universal links, so my vote would be to leave it in.